scripts/docgen.mjs

```diff
@@ -148,10 +148,26 @@ function getDeprecatedAside(tag) {
 }
 
 /**
- * Get block of type, CLI command, default value, and version added in for the current comment.
+ * Retrieve the Meta component props as a string from the given data.
+ * @param {{ cli?: string; defaultValue?: string; version?: string }} data
+ */
+function getMetaComponentProps(data) {
+	const props = [];
+	if (data.cli) props.push('cli="`' + data.cli + '`"');
```

Comment:
It seems weird to have hard-coded backticks surrounding this value just for it to be replaced inside of the component. Wouldn't it be better for the component to wrap anything passed in this field with `<code>`?

```suggestion
	if (data.cli) props.push(`cli="${data.cli}"`);
```

Then on the component:
```diff
- <dd class="meta-value" set:html={replaceBackticksWithCodeTags(cli)} />
+ <dd class="meta-value"><code set:html={cli}></code></dd>